### DISCUSSION TITLE
6% of requests to Nova-2 fail with 503

### DISCUSSION THREAD

**Discussion by kpetrovsky-audibene** [deepgram/community#906] [General help] [answered]
Hi,
We have a consistent issue when using Deepgram: nova2 model, English, pre-recorded transcription. Recording duration is in the range from 10 to 80 minutes.
For some period of time (e.g. 1-2 days), all requests will be processed without a problem. And then we'll hit a patch of time (30-60 mins) where 50% of requests will fail with a 503 error.
I checked the average error rate for 4 days, and it was at 6%. Are we doing something wrong?
This is a big blocker for us, we can't build a solid end user experience with a 6% error rate, and it also doesn't inspire confidence in the realtime transcription.

**Some numbers**
Time range: 17-21 of August
Total requests: 513
503 errors: 31

**Details**
Request format: `/v1/listen?model=nova-2&smart_format=true&multichannel=true&diarize=true&language=en&numerals=true&encoding=linear16&callback=hidden&extra=hidden&extra=hidden`

**Examples**
Successful request: `19da215a-9d92-44e1-a557-b4285581bcc5`
1.5 minutes later, an error: `d036ca11-ee75-43f7-afe3-baa53ce75ef2`
6.5 minutes later, still an error: `97a14dc2-5bd1-44cd-8731-623a96176c3b`
50 seconds later - working again: `c812cb25-387a-4fbd-9def-4f300afba406`

Any ideas/recommendations?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
Hi @kpetrovsky-audibene 

Just want to acknowledge this here. We are discussing in Discord right now.

**Comment by davidvonthenen** [comment, marked as answer]
Closing the loop on this, speaking with the individual in Discord, removing the encoding value (probably using containerized audio) and adding a sample rate dropped the error rate to ~0.1%.
